bigbluebutton-html5/imports/ui/components/LoadingScreen/styles.scss

```diff
@@ -0,0 +1,65 @@
+
+:root {
+  --loader-bg: #FFF;
+  --loader-bullet: #8B9AA8;
+  --loader-message-color: #000000;
+}
+
+.background {
+  position: fixed;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  background-color: var(--loader-bg);
+}
+
+.message {
+  font-size: 20px;
+  color: var(--loader-message-color);
+  text-align: center;
+}
+
+.spinner {
+  width: 100%;
+  text-align: center;
+  height: 22px;
+  margin-bottom: 16px;
+}
+
+.spinner > div {
+  width: 18px;
+  height: 18px;
+  margin: 0 5px;
+  background-color: var(--loader-bullet);
+  border-radius: 100%;
+  display: inline-block;
+
+  :global(.animationsEnabled) & {
+    animation: sk-bouncedelay 1.4s infinite ease-in-out both;
+  }
+}
+
+.spinner .bounce1 {
+  :global(.animationsEnabled) & {
+    animation-delay: -0.32s;
+  }
+}
+
+.spinner .bounce2 {
+  :global(.animationsEnabled) & {
+    animation-delay: -0.16s;
+  }
+}
+
+@keyframes sk-bouncedelay {
+  0%,
+  80%,
+  100% {
+    transform: scale(0);
+  }
+  40% {
+    transform: scale(1.0);
+  }
+}
```

Comment:
here it requires EOF should be empty line

Reply:
added